### ISSUE TITLE
Add the missing descriptive names to Show App Details page

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#215]
@krozett The following are missing descriptive names on Show App Details page
a.	Rating score radio buttons
b.	App rating review textbox
![image](https://user-images.githubusercontent.com/1034174/108133926-3308ef80-706a-11eb-8c7a-e08e1c7854d6.png)

Accessibility success criterion:
* https://www.w3.org/TR/WCAG21/#non-text-content
* https://www.w3.org/TR/WCAG21/#label-in-name

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
